pkg/utils/machines.go

```diff
@@ -25,3 +32,19 @@ func GetMasterMachines(kclient client.Client) (*machineapi.MachineList, error) {
 	}
 	return machineList, nil
 }
+
+func GetControlPlaneMachineSet(kclient client.Client) (*machinev1.ControlPlaneMachineSet, error) {
+	cpms := &machinev1.ControlPlaneMachineSet{}
```

Comment:
We would need to register `machinev1` to the scheme in `main.go` as well

Reply:
Good call - I think that got overwritten in the rebase.